lib/vapi/DeployedModels.py

```diff
@@ -23,7 +23,7 @@ class DeployedModels:
     def __init__(self, server):
         self.server = server
 
-    def create(self, modelid, name, **kwargs):
+    def create(self, modelid, datasetid, name, **kwargs):
```

Comment:
`datasetid` should be part of the `kwargs` and use the same name as the json field that is sent in the api request.